src/djimatrice_drone.py

```diff
@@ -0,0 +1,230 @@
+import roslibpy
+import numpy as np
+from drone import Drone
+from enum import Enum
+
+class DjiMatriceDrone(Drone):
+
+    drone_type = "DjiMatrice"
+
+    class DroneTaskControl(Enum):
+        GO_HOME = 1
+        TAKEOFF = 4
+        LANDING = 6
+
+    def __init__(self, drone_name, drone_type, id=False):
+        super().__init__(drone_name, drone_type, id)
+        assert(drone_type == self.drone_type)
+        self.prev_flight_status = Drone.Flight_Status.NULL
+
+    # TODO Implement
+    def upload_mission(self, waypoints):
+        self.waypoints = waypoints
+        self.mission_msg_list = []
+        if self.flight_status == Drone.Flight_Status.ON_GROUND_STANDBY:
+            for i in range(len(self.waypoints)):
+                way_point_msg = waypoints[i]
+                self.mission_msg_list.append(way_point_msg)
+            way_point_task = way_point_msg
+            self.upload_waypoint_task(way_point_task)
```

Comment:
We should make all our spelling of waypoint consistent, so change way_point to waypoint in the variable names.